### PR TITLE
Add Authorization

### PR DESCRIPTION
**Pull request by lindit**

### CODE REVIEW

**Comment by lindit** [comment]
- [ ] [Merge Management-Service first](https://github.com/admin-ch/CovidCertificate-Management-Service/pull/323)

File: src/main/java/ch/admin/bag/covidcertificate/api/exception/AuthorizationException.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package ch.admin.bag.covidcertificate.api.exception;
+
+import lombok.Getter;
+import org.springframework.core.NestedRuntimeException;
+
+/**
+ * Falls eine Funktion nicht konfiguriert ist, muss diese im
+ * bzw. im application-authorization.yml konfiguriert werden.
+ */
```
Comment (mofobo-bit) [inline src/main/java/ch/admin/bag/covidcertificate/api/exception/AuthorizationException.java:9]: I'm not sure but is this comment complete / correct ?

File: src/main/java/ch/admin/bag/covidcertificate/config/security/authentication/JeapAuthenticationToken.java [hunk 2]
```diff
@@ -27,7 +27,7 @@ public JeapAuthenticationToken(Jwt jwt, Set<String> userRoles) {
      * @return The client id specified in this token.
      */
     public String getClientId() {
-        return getToken().getClaimAsString("clientId");
+        return getToken().getClaimAsString("client_id");
```
Comment (mofobo-bit) [inline src/main/java/ch/admin/bag/covidcertificate/config/security/authentication/JeapAuthenticationToken.java:30]: Did this previously work?

**Review by mofobo-bit — APPROVED** [review]

File: src/main/java/ch/admin/bag/covidcertificate/config/security/authentication/JeapAuthenticationToken.java (hunk 2 above)
Comment (lindit) [reply src/main/java/ch/admin/bag/covidcertificate/config/security/authentication/JeapAuthenticationToken.java:30]: Was never used. I needed to change it a few weeks ago in Management-Service to make it work.

**Review by haraldloesing — APPROVED** [review]

**Pull request merged**